docs/experience/url-groups.md

```diff
@@ -0,0 +1,39 @@
+title: URL Groups
+description: Guide for using URL Groups in Experience
```

Comment:
description: Guide for using URL Groups in Experience
->
description: A guide for using URL Groups in Sematext Experience. Group routes with different parameters to gain a better overview of performance data. Creating URL Groups is simple and include setting custom time thresholds for each group.